src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,22 @@
+package duke.task;
+
+public class ToDo extends Task {
+
+    /**
+     * This is a constructor for the ToDo object.
+     *
+     * @param description The description of the ToDo
+     */
+    public ToDo(String description) {
+        super(description);
+        this.taskType = TaskType.T;
+    }
+
```

Comment:
I recommend you Override the getDataString Method here and simply call the super.getDataString.

I know it may seem redundant but I think it is a better practice.

Reply:
Cool! Will proceed to implement (: